diff -Naur a/etc/skel/om-welcome.desktop b/etc/skel/om-welcome.desktop
--- a/etc/skel/om-welcome.desktop	2014-04-26 17:58:29.000000000 +0000
+++ b/etc/skel/om-welcome.desktop	2014-04-27 15:49:59.000000000 +0000
@@ -1,10 +1,10 @@
 [Desktop Entry]
-GenericName=Welcome to OpenMandriva
-Name=om-welcome
+Comment=Welcome to OpenMandriva
+Name=Welcome to OpenMandriva
 Exec=/usr/bin/om-welcome
 X-KDE-StartupNotify=true
 X-KDE-HasTempFileOption=true
 Icon=openmandriva
 Type=Application
 Terminal=false
-Categories=Qt;KDE;
+Categories=Qt;KDE;Settings;
diff -Naur a/om-welcome.desktop b/om-welcome.desktop
--- a/om-welcome.desktop	2014-04-26 17:58:29.000000000 +0000
+++ b/om-welcome.desktop	2014-04-27 15:49:59.647481678 +0000
@@ -1,10 +1,10 @@
 [Desktop Entry]
-GenericName=Welcome to OpenMandriva
-Name=om-welcome
-Exec=/usr/bin/om-welcome-launcher
+Comment=Welcome to OpenMandriva
+Name=Welcome to OpenMandriva
+Exec=/usr/bin/om-welcome
 X-KDE-StartupNotify=true
 X-KDE-HasTempFileOption=true
 Icon=openmandriva
 Type=Application
 Terminal=false
-Categories=Qt;KDE;
+Categories=Qt;KDE;Settings;
