diff -Naur oma-welcome-1.0.5/usr/bin/om-welcome oma-welcome-1.0.5.tpg/usr/bin/om-welcome
--- oma-welcome-1.0.5/usr/bin/om-welcome	2014-09-23 15:54:48.000000000 +0000
+++ oma-welcome-1.0.5.tpg/usr/bin/om-welcome	2015-01-19 19:37:19.069451932 +0000
@@ -1,3 +1,3 @@
 #!/bin/bash
 cd /usr/share/oma-welcome/;
-python bigbashview.py -s 870x520 -c -i /usr/share/icons/openmandriva.svg index.sh.htm 2> /dev/null;
+python2 bigbashview.py -s 870x520 -c -i /usr/share/icons/openmandriva.svg index.sh.htm 2> /dev/null;
diff -Naur oma-welcome-1.0.5/usr/bin/om-welcome-launcher oma-welcome-1.0.5.tpg/usr/bin/om-welcome-launcher
--- oma-welcome-1.0.5/usr/bin/om-welcome-launcher	2014-09-23 15:54:48.000000000 +0000
+++ oma-welcome-1.0.5.tpg/usr/bin/om-welcome-launcher	2015-01-19 19:37:14.171320311 +0000
@@ -13,6 +13,6 @@
 if [ $autostart -eq 1 ]; then
 
 	cd /usr/share/oma-welcome/;
-	python bigbashview.py -s 870x520 -c -i /usr/share/icons/openmandriva.svg index.sh.htm 2> /dev/null;
-		
+	python2 bigbashview.py -s 870x520 -c -i /usr/share/icons/openmandriva.svg index.sh.htm 2> /dev/null;
+
 fi
