diff -Naur oma-welcome-1.0.3/Makefile oma-welcome-1.0.3.tpg/Makefile
--- oma-welcome-1.0.3/Makefile	2014-04-27 18:50:51.000000000 +0000
+++ oma-welcome-1.0.3.tpg/Makefile	2014-04-29 18:41:20.915325360 +0000
@@ -1,21 +1,29 @@
-NAME=oma-welcome
-VERSION=1.0.3
-
-all:
-
-install:
-	mkdir -p $(DESTDIR)$(prefix)/$(bindir)
-	mkdir -p $(DESTDIR)$(prefix)/$(sysconfdir)/skel/Desktop
-	mkdir -p $(DESTDIR)$(prefix)/$(sharedir)/$(NAME)
-	mkdir -p $(DESTDIR)$(prefix)/$(sharedir)/applications
-	mkdir -p $(DESTDIR)$(prefix)/$(localedir)
-	install -m 644 data/*.desktop $(DESTDIR)$(prefix)/$(sysconfdir)/skel/Desktop
-	install -m 644 data/*.desktop $(DESTDIR)$(prefix)/$(sharedir)/applications
-	install -m 755 usr/bin/* $(DESTDIR)$(prefix)/$(bindir)
-	install -m 644 usr/share/om-welcome/* $(DESTDIR)$(prefix)/$(sharedir)/$(NAME)
-	install -m 644 usr/share/locale/* $(DESTDIR)$(prefix)/$(localedir)
-
-dist:
-	git archive --format=tar --prefix=$(NAME)-$(VERSION)/ HEAD | xz -2vec -T0 > $(NAME)-$(VERSION).tar.xz;
-	$(info $(NAME)-$(VERSION).tar.xz is ready)
-
+NAME=oma-welcome
+VERSION=1.0.3
+TRANSLATIONS=en fr id it pt_BR tr
+bindir=/usr/bin
+sysconfdir=/etc
+sharedir=/usr/share
+localedir=$(sharedir)/locale
+
+all:
+
+install:
+	mkdir -p $(DESTDIR)$(prefix)/$(bindir)
+	mkdir -p $(DESTDIR)$(prefix)/$(sysconfdir)/xdg/autostart
+	mkdir -p $(DESTDIR)$(prefix)/$(sharedir)/$(NAME)
+	mkdir -p $(DESTDIR)$(prefix)/$(sharedir)/applications
+	mkdir -p $(DESTDIR)$(prefix)/$(localedir)
+	install -m 644 om-welcome.desktop $(DESTDIR)$(prefix)/$(sysconfdir)/xdg/autostart
+	install -m 644 etc/skel/om-welcome.desktop $(DESTDIR)$(prefix)/$(sharedir)/applications
+	install -m 755 usr/bin/* $(DESTDIR)$(prefix)/$(bindir)
+	cp -avx usr/share/$(NAME)/* $(DESTDIR)$(prefix)/$(sharedir)/$(NAME)
+	@for l in $(TRANSLATIONS); do \
+	mkdir -p  $(DESTDIR)$(prefix)/$(localedir)/$$l/LC_MESSAGES; \
+	cp -avx locale/$$l/* $(DESTDIR)$(prefix)$(localedir)/$$l/LC_MESSAGES ; \
+	done
+
+dist:
+	git archive --format=tar --prefix=$(NAME)-$(VERSION)/ HEAD | xz -2vec -T0 > $(NAME)-$(VERSION).tar.xz;
+	$(info $(NAME)-$(VERSION).tar.xz is ready)
+
